WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItemOriginCountryInfoDialog.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+
+struct WooShippingCustomsItemOriginCountryInfoDialog: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    @Environment(\.dismiss) var dismiss
+
+    var body: some View {
+        ZStack {
+            Color.black.opacity(Layout.backgroundOpacity).edgesIgnoringSafeArea(.all)
+
+                VStack {
+                    GeometryReader { geometry in
+                        ScrollView {
+                            VStack(alignment: .center, spacing: Layout.verticalSpacing) {
+                                Text(Localization.title)
+                                    .headlineStyle()
+                                Text(Localization.bodyParagraph)
+                                    .bodyStyle()
+                                    .fixedSize(horizontal: false, vertical: true)
+                                Button {
+                                    dismiss()
+                                } label: {
+                                    Text(Localization.doneButtonTitle)
+                                }
+                                .buttonStyle(PrimaryButtonStyle())
+                            }
+                            .padding(Layout.outterPadding)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .background(Color(.systemBackground))
+                            .cornerRadius(Layout.cornerRadius)
+                            .frame(width: geometry.size.width)
+                            .frame(minHeight: geometry.size.height)
+                        }
+                }
+            }
+            .padding(Layout.outterPadding)
+            .frame(maxWidth: .infinity, alignment: .center)
+        }
+    }
+}
+extension WooShippingCustomsItemOriginCountryInfoDialog {
+    enum Localization {
+        static let title = NSLocalizedString("Origin Country", comment: "Title for the custom origin country educational dialog")
+        static let bodyParagraph = NSLocalizedString("Country where the product was manufactured or assembled.",
+                                                          comment: "Body for the custom items origin country educational dialog")
+        static let doneButtonTitle = NSLocalizedString("Done",
+                                                       comment: "Button title for the done button in the customs description educational dialog")
```

Comment:
Same as above, these strings need the reverse-DNS keys per our updated localization guidelines.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/commit/58f965c70ffbfe4a2bc37ef0983d62045d256977